src/test/java/org/jenkinsci/plugins/github/test/GitHubServerConfigMatcher.java

```diff
@@ -31,4 +84,50 @@ protected String featureValueOf(GitHubServerConfig actual) {
             }
         };
     }
+
+    public static Matcher<GitHubServerConfig> withIsManageHooks(Matcher<Boolean> matcher) {
+        return new FeatureMatcher<GitHubServerConfig, Boolean>(matcher, "is manage hooks", "") {
+            @Override
+            protected Boolean featureValueOf(GitHubServerConfig actual) {
+                return actual.isManageHooks();
+            }
+        };
+    }
+
+    public static Matcher<Mapping> withIsManageHooksS(Matcher<Boolean> matcher) {
+        return new FeatureMatcher<Mapping, Boolean>(matcher, "is manage hooks", "") {
+            @Override
+            protected Boolean featureValueOf(Mapping actual) {
+                return Boolean.valueOf(valueOrNull(actual, "manageHooks"));
+            }
+        };
+    }
+
+    public static Matcher<GitHubServerConfig> withName(Matcher<String> matcher) {
+        return new FeatureMatcher<GitHubServerConfig, String>(matcher, "name", "") {
+            @Override
+            protected String featureValueOf(GitHubServerConfig actual) {
+                return actual.getName();
+            }
+        };
+    }
+
+    public static Matcher<Mapping> withNameS(Matcher<String> matcher) {
+        return new FeatureMatcher<Mapping, String>(matcher, "name", "") {
+            @Override
+            protected String featureValueOf(Mapping actual) {
+                return valueOrNull(actual, "name");
+            }
+        };
+    }
+
+    private static String valueOrNull(Mapping mapping, String key) {
+        try {
+            return mapping.get(key).asScalar().getValue();
+        } catch (NullPointerException | ConfiguratorException e) {
+//            LOG.error("", e);
```

Comment:
?

Reply:
There is no methods to check existance of the key. Either get once with removal either get NPE. Exception is expected against object that has no key, no need to log, so commented.

Reply:
more around is this commented out code intended to stay.